src/main/resources/org/jenkinsci/plugins/envinject/EnvInjectBuildWrapper/help-propertiesContent.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
@oleg-nenashev We need both help files because we can use this plugin in different ways:

* Prepare an environment for the run: *...set an environment for the build job* (before)
* Inject environment variables to the build process: *... to the whole build process.* (during)

Please, find attached this screenshot:

![jenkins-29817](https://cloud.githubusercontent.com/assets/1021745/9244568/a91ebc8e-419c-11e5-911d-31cc2ab95e79.png)